pinot-common/src/main/java/org/apache/pinot/common/function/scalar/ArrayFunctions.java

```diff
@@ -347,4 +347,9 @@ public static double[] generateDoubleArray(double start, double end, double inc)
     }
     return arr;
   }
+
+  @ScalarFunction
+  public static String arrayJoinString(String[] values, String delimiter) {
```

Comment:
Is there a Postgres equivalent? Does it make sense to use the same/similar name?

Reply:
Postgres has a function called: `ARRAY_TO_STRING` : https://www.postgresql.org/docs/9.1/functions-array.html

I think we can implement this signature.

Reply:
Can you change the function name to `arrayToString`

Reply:
For null handling you need to create a new function:
```
public static String arrayToString(String[] values, String delimiter, String nullString) {
...
```

Reply:
@xiangfu0 done. PTAL - need your eyes on the null checker.